files/initramfs-tools/union-mount.j2

```diff
@@ -17,8 +17,10 @@ set_tmpfs_log_partition_size()
 
   # NOTE: certain platforms, when reaching initramfs stage, have a small
   #       limit of mounting tmpfs partition, potentially due to amount
-  #       of RAM available in this stage. e.g. Arista 7050 QX32
+  #       of RAM available in this stage. e.g. Arista 7050i-qx32[s] and 7060-cx32s
```

Comment:
Not to pick nits, but you now have a stray "i" (7050i).

Reply:
brought to you by your friendly neighborhood vi editor
corrected